lib/widgets/message_list.dart

```diff
@@ -1195,13 +1195,15 @@ class _MessageListState extends State<MessageList> with PerAccountStoreAwareStat
           key: ValueKey(data.message.id),
           narrow: widget.narrow,
           header: header,
-          item: data);
+          item: data,
+          isLastInFeed: isLastInFeed);
```

Comment:
nit: metadata before payload / `child`-like argument (so here, isLastInFeed before header and item)